doc/libpmem2/pmem2_map.3.md

```diff
@@ -43,9 +43,10 @@ of the calling process. This function requires a configuration
 If the **pmem2_map**() function succeeds in creating a new mapping it
 instantiates a new *struct pmem2_map** object describing the mapping. The
 pointer to this newly created object is stored in the user-provided variable
-passed via the *map_ptr* pointer. If the mapping fails the variable pointed by
-*map_ptr* will contain a NULL value and appropriate error value will be
-returned. For a list of possible return values please see
+passed via the *map_ptr* pointer. On Windows the *map_ptr* pointer requires
```

Comment:
That does that mean? A pointer cannot require an open file descriptor. What does it mean to "flush buffers"?